crates/cubecl-wgpu/tests/constant_array.wgsl

```diff
@@ -0,0 +1,28 @@
+@group(0)
+@binding(0)
+var<storage, read_write> output_0_global: array<f32>;
+
+@group(0)
+@binding(1)
+var<storage, read_write> info: array<u32>;
+
+const arrays_0: array<f32, 3> = array(f32(3u),f32(5u),f32(1u),);
```

Comment:
Looks better in wgsl

Reply:
`wgsl` uses the `fmt_cast` function I introduced in the variable allocator PR to add casts where necessary. CUDA doesn't have that rn.